chaindexing-postgres/tests/repo.test.ts

```diff
@@ -14,19 +14,25 @@ describe('Repo', async () => {
     await repo.migrate(rootConn, postgresRepoMigrations.getInternalMigrations());
   });
 
-  describe('createContractAddresses', async () => {
+  describe.only('createContractAddresses', async () => {
     it('saves unsaved contract addresses', async (conn) => {
       const unsavedContractAddresses = UnsavedContractAddressFactory.manyNew(2).toSorted();
       await repo.createContractAddresses(conn, unsavedContractAddresses);
 
-      await repo.streamContractAddresses(conn, (contractAddresses) => {
-        const contractAddressesSorted = contractAddresses.toSorted();
+      const streamedAddresses = repo.streamContractAddresses(conn);
```

Comment:
Nit: Naming suggestion:

```suggestion
      const contractAddressStream = repo.getContractAddressesStream(conn);
```